policy/modules/services/container.te

```diff
@@ -44,6 +44,14 @@ gen_tunable(container_use_nfs, false)
 ## </desc>
 gen_tunable(container_use_samba, false)
 
+## <desc>
+##      <p>
+##      Allow containers to use eCryptfs filesystems.
+##      </p>
+## </desc>
+gen_tunable(container_use_ecryptfs, false)
+
+
```

Comment:
Please move this declaration above `container_use_nfs` so the declarations are in order.